src/System.Diagnostics.Process/tests/ProcessThreadTests.cs

```diff
@@ -14,6 +14,7 @@ namespace System.Diagnostics.Tests
     public class ProcessThreadTests : ProcessTestBase
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapNotUapAot, "https://github.com/dotnet/corefx/issues/22174")]
```

Comment:
It is not disabled test but skipped test. I agree mentioning the issue # like this will be confusing.